example/lib/widgets/order_sheet.dart

```diff
@@ -147,6 +147,8 @@ class OrderSheet extends StatelessWidget {
           CookieButton(
             text: "Buyer Verification",
             onPressed: () {
+
+              Scaffold.of(context).showBottomSheet((context) =>Container());
```

Comment:
is this change related to the SDK upgrades?

Reply:
No, I've removed it's not related for SDK upgrade and i've upgraded PR.